tests/test_files/chat_with_website.yaml

```diff
@@ -1,50 +0,0 @@
-components:
```

Comment:
why are we removing this file?

Reply:
@anakin87 as discussed, I've just added and tested the working example of `chat_with_website` pipeline